tasks.py.jinja

```diff
@@ -283,13 +284,15 @@ def lint(c, verbose=False):
 
 
 @task(develop)
-def start(c, detach=True, debugpy=False):
+def start(c, detach=True, debugpy=False, sleep=0):
     """Start environment."""
     cmd = "docker-compose up"
     if detach:
         cmd += " --detach"
     with c.cd(str(PROJECT_ROOT)):
         c.run(cmd, env=dict(UID_ENV, DOODBA_DEBUGPY_ENABLE=str(int(debugpy))))
+    if sleep:
+        time.sleep(sleep)
```

Comment:
This is usually not a good idea, because you will slow down fast machines, or you will have the same bug in slow machines.

It's better to query the port and unblock the task when it responds as listening.

Reply:
Ok. Thanks for the suggestion. I will work on that.